File: Programming Basics/C++/Introduction/Repainting/Repainting.cpp
#include <iostream>

int main()
{
    int nylon, paint, diluent, hours;
    std::cin >> nylon >> paint >> diluent >> hours;

    double materialsCost = 1.5 * (nylon + 2) + 14.5 * (paint * 1.1) + diluent * 5 + 0.4;
    double wage = 0.3 * materialsCost;
    double total = materialsCost + wage * hours;

    std::cout << total << std::endl;
}

File: Programming Basics/JavaScript/Introduction/Exercises/repainting.js
function solve(input) {
    const nylon = Number(input[0]);
    const paint = Number(input[1]);
    const diluent = Number(input[2]);
    const hours = Number(input[3]);

    const materialsCost = 1.5 * (nylon + 2) + 14.5 * (paint * 1.1) + diluent * 5 + 0.4;
    const wage = 0.3 * materialsCost;

    const total = materialsCost + wage * hours;
    console.log(total);
}
